src/api/forms/service.js

```diff
@@ -335,7 +336,8 @@ export async function createLiveFromDraft(formId, author) {
       throw Boom.badRequest(makeFormLiveErrorMessages.missingStartPage)
     }
 
-    if (!draftFormDefinition.outputEmail) {
+    if (!draftFormDefinition.outputEmail && !form.notificationEmail) {
```

Comment:
Is it definitely just the one that's needed or is it both?

```suggestion
    if (!draftFormDefinition.outputEmail || !form.notificationEmail) {
```